articles/metadata/rules.md

```diff
@@ -51,19 +47,17 @@ function(user, context, callback){
   if (user.user_metadata.preferences.color === 'black'){
     // code to be executed
   }
-
   ...
-
 }
 ```
 
 ## Updating Metadata
 
-All [rules](/rules) have available an `auth0` object (which is an instance of the [node-auth0 SDK](https://github.com/auth0/node-auth0) that is capable of calling the Auth0 Management API v2). The `auth0` object is preconfigured with the necessary permissions to update users.
+All rules have available an `auth0` object (which is an instance of the [node-auth0 SDK](https://github.com/auth0/node-auth0)) that is capable of calling the Auth0 Management API v2. The `auth0` object is preconfigured with the necessary permissions to update users.
```

Comment:
I think you could drop the 'available' from the first sentence. `All rules have an .. `